content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst

```diff
@@ -2,35 +2,46 @@
 Invoice Time Spent on Tickets (Postpaid Support Services)
 =========================================================
 
-Directly pull the billable time you have tracked on your helpdesk tickets into sales orders and
-invoices through a project task. It gives you more control over what you charge your client, and it
-is more efficient.
+Connect your Helpdesk ticket to a Project task and use either one to record timesheets that you can later pull from to bill your client. This gives you more control over what you charge your client, and it is more efficient.
 
 Configuration
 =============
+Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams` and select the Helpdesk team you want to track timesheets for. Click **Edit** and enable the options Timesheet on Ticket and Time Reinvoicing.
```

Comment:
Italicize Odoo options to indicate that these are choices and features of the software.

```suggestion
Click **Edit** and enable the options *Timesheet on Ticket* and *Time Reinvoicing*.
```